modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/portlet/action/test/UnbindObjectDefinitionMVCResourceCommandTest.java

```diff
@@ -0,0 +1,313 @@
+				_objectRelationshipLocalService.getObjectRelationship(
+					objectRelationship.getObjectRelationshipId());
+
+			Assert.assertEquals(edge, objectRelationship.isEdge());
+		}
+	}
+
+	private void _assertRootObjectDefinitionId(
+			long rootObjectDefinitionId, ObjectDefinition... objectDefinitions)
+		throws Exception {
+
+		for (ObjectDefinition objectDefinition : objectDefinitions) {
+			objectDefinition =
+				_objectDefinitionLocalService.getObjectDefinition(
+					objectDefinition.getObjectDefinitionId());
+
+			Assert.assertEquals(
+				rootObjectDefinitionId,
+				objectDefinition.getRootObjectDefinitionId());
+		}
+	}
+
+	private ObjectDefinition _createObjectDefinition(
+			String objectDefinitionName)
+		throws Exception {
+
+		ObjectDefinition objectDefinition =
+			_objectDefinitionLocalService.addCustomObjectDefinition(
+				TestPropsValues.getUserId(), 0, false, false,
+				LocalizedMapUtil.getLocalizedMap(objectDefinitionName),
+				objectDefinitionName, null, null,
+				LocalizedMapUtil.getLocalizedMap(RandomTestUtil.randomString()),
+				false, ObjectDefinitionConstants.SCOPE_COMPANY,
+				ObjectDefinitionConstants.STORAGE_TYPE_DEFAULT,
+				Collections.singletonList(
+					new TextObjectFieldBuilder(
+					).labelMap(
+						LocalizedMapUtil.getLocalizedMap(
+							RandomTestUtil.randomString())
+					).name(
+						StringUtil.randomId()
+					).build()));
+
+		return _objectDefinitionLocalService.updateRootObjectDefinitionId(
+			objectDefinition.getObjectDefinitionId(),
+			_objectDefinitionA.getObjectDefinitionId());
+	}
+
+	private ObjectRelationship _relateObjectDefinition(
+			ObjectDefinition objectDefinition1,
+			ObjectDefinition objectDefinition2)
+		throws Exception {
+
+		ObjectRelationship objectRelationship =
+			_objectRelationshipLocalService.addObjectRelationship(
+				TestPropsValues.getUserId(),
+				objectDefinition1.getObjectDefinitionId(),
+				objectDefinition2.getObjectDefinitionId(), 0,
+				ObjectRelationshipConstants.DELETION_TYPE_CASCADE,
+				LocalizedMapUtil.getLocalizedMap(
+					objectDefinition1.getShortName() + " - " +
+						objectDefinition2.getShortName()),
+				StringUtil.randomId(),
+				ObjectRelationshipConstants.TYPE_ONE_TO_MANY);
+
+		return _objectRelationshipLocalService.updateObjectRelationship(
+			objectRelationship.getObjectRelationshipId(), 0,
+			ObjectRelationshipConstants.DELETION_TYPE_CASCADE, true,
+			objectRelationship.getLabelMap());
+	}
+
+	private String _toString(Tree tree) {
+		Iterator<Node> iterator = tree.iterator();
+
+		StringBundler sb = new StringBundler(2);
+
+		iterator.forEachRemaining(
+			node -> {
+				if (!node.isRoot()) {
+					sb.append(", ");
+				}
+
+				ObjectDefinition objectDefinition =
+					_objectDefinitionLocalService.fetchObjectDefinition(
+						node.getObjectDefinitionId());
+
+				sb.append(objectDefinition.getShortName());
+			});
+
+		return sb.toString();
+	}
+
+	private void _unbindObjectDefinition(long objectDefinitionId)
```

Comment:
@MarinhoFeliphe rename to `_unbind`

Reply:
done [here](https://github.com/liferay-objects/liferay-portal/pull/2458/commits/0daa57981a1e5533cbc4d4cbe74be7ee030cd1f9)